src/main/java/sirius/biz/tenants/TenantUserManager.java

```diff
@@ -319,7 +319,12 @@ public UserInfo findUserByUserId(String accountId) {
     @Override
     public UserInfo bindToRequest(@Nonnull WebContext ctx) {
         UserInfo info = super.bindToRequest(ctx);
-        return verifyIpRange(ctx, info);
+        if (info.isLoggedIn()) {
+            // We only need to verify the IP range for users being logged in....
```

Comment:
I am a little bit confused:.
in the comment is "being logged in" but you check on "info.isLoggedIn()"
are we checking if the user is being logged in in this request or if the user is already logged in?